src/bitmessagekivy/mpybit.py

```diff
@@ -206,6 +206,80 @@ def load_screen_callback(self, instance, dt=0):
             except Exception as e:
                 self.root.ids.id_trash.children[0].children[1].active = False
 
+    @staticmethod
+    def get_current_address(current_add_label, current_addr):
```

Comment:
should only return the selected item. No arguments, no access `config`, no string contatenation, ....